services/users/constants.go

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package users
+
+const MissingAccountError = "app.user.missing_account.const"
```

Comment:
We should change these to a normal error - `var MissingAccountError = errors.New("...")`